containers/src/Data/Sequence/Internal.hs

```diff
@@ -2401,7 +2429,11 @@ scanl f z0 xs = z0 <| snd (mapAccumL (\ x z -> let x' = f x z in (x', x')) z0 xs
 -- __Note__: This function is partial.
 --
 -- > scanl1 f (fromList [x1, x2, ...]) = fromList [x1, x1 `f` x2, ...]
+#if __GLASGOW_HASKELL__ >= 800
+scanl1 :: HasCallStack => (a -> a -> a) -> Seq a -> Seq a
```

Comment:
I'm concerned this could build a call stack recursively. Check the Core, please.